components/expenses/Transaction.js

```diff
@@ -141,9 +149,20 @@ class Transaction extends React.Component {
         <Container ml={3} width={1}>
           <Flex justifyContent="space-between" alignItems="baseline">
             <div>
-              <P fontSize="1.4rem" color="#313233" display="inline">
-                {description}
-              </P>
+              {(expense && (
+                <Link
+                  route={`/${collective.slug}/expenses/${expense.id}`}
+                  fontSize="1.4rem"
```

Comment:
`Link` component doesn't support the `fontSize`, `color` and `display` props.

It's a good thing to keep the existing styles, so a possible solution I could recommend would be to just wrap the previous `<P ...>...</P>` component in the link that you added:

```jsx
<Link route={`/${collective.slug}/expenses/${expense.id}`}>
  <P fontSize="1.4rem" color="#313233" display="inline">{description}</P>
</Link>
```